src/CommonGLPI.php

```diff
@@ -627,7 +627,7 @@ public static function displayStandardTab(CommonGLPI $item, $tab, $withtemplate
                 if (count($ong)) {
                     foreach ($ong as $key => $val) {
                         if ($key != 'empty') {
-                            echo "<div class='alltab'>$val</div>";
+                            echo "<div class='alltab'>" . htmlspecialchars($val) . "</div>";
```

Comment:
Unless I miss something, `$val` will contains some HTML code:
```php
    public function addDefaultFormTab(array &$ong)
    {
        // ...
        $icon = $icon ? "<i class='$icon me-2'></i>" : '';
        $ong[static::getType() . '$main'] = '<span>' . $icon . static::getTypeName(1) . '</span>';
        // ...
    }
```
```suggestion
                            echo "<div class='alltab'>" . $val . "</div>";
```